src/main/java/io/curity/identityserver/plugin/github/authentication/CallbackRequestHandler.java

```diff
@@ -180,7 +177,7 @@ private WebServiceClient getWebServiceClient(String uri)
                                 requiredScheme, configuredScheme));
             }
 
-            return _webServiceClientFactory.create(h).withHost(u.getHost());
+            return _webServiceClientFactory.create(h).withHost(u.getHost()).withPath(u.getPath());
```

Comment:
ah, yes. This was a bug in this plug-in. Whew. SDK is fine and no unexpected breaking change :)